apps/ledger-live-desktop/src/renderer/modals/Send/fields/AmountField.tsx

```diff
@@ -14,16 +12,16 @@ import Switch from "~/renderer/components/Switch";
 import Text from "~/renderer/components/Text";
 
 type Props = {
-  parentAccount: ?Account,
-  account: AccountLike,
-  transaction: Transaction,
-  onChangeTransaction: (*) => void,
-  status: TransactionStatus,
-  bridgePending: boolean,
-  t: TFunction,
-  initValue?: BigNumber,
-  walletConnectProxy?: boolean,
-  resetInitValue?: () => void,
+  parentAccount?: Account;
```

Comment:
Is it your prettier that did those changes automatically ?

Reply:
No I migrated the file to typescript

Reply:
Oh so it was a tsx file using flow as typecheck D:

Reply:
Ok let's be careful and make sure it works with the generated component mechanic

Reply:
sync-families-dispatch.mjs